client/packages/common/src/utils/pricing/PricingUtils.ts

```diff
@@ -5,11 +5,9 @@ export const PricingUtils = {
     return Math.max(total - subtotal, 0);
   },
   effectiveTax: (subtotal: number, total: number) => {
-    const taxAmount = PricingUtils.taxAmount(subtotal, total);
-    const x =
-      Math.round(
-        (taxAmount / Math.max(subtotal, 1)) * 100 * ROUNDING_PRECISION
-      ) / ROUNDING_PRECISION;
-    return x;
+    return (
+      Math.round(((total - subtotal) / subtotal) * 100 * ROUNDING_PRECISION) /
+      ROUNDING_PRECISION
+    );
```

Comment:
I think this must be doing something spicy? 

When no service charges:
![Screenshot 2024-07-02 at 11 46 09 AM](https://github.com/msupply-foundation/open-msupply/assets/55115239/5983b304-9c5b-4530-9cdc-d4ae1f400331)

Once you add service charges, above looks fine, but tax edit modal does this:
![Screenshot 2024-07-02 at 11 48 02 AM](https://github.com/msupply-foundation/open-msupply/assets/55115239/75525a28-d681-4023-b513-ea1337bf83ae)

Reply:
Using Math.max for subtotal is returning some wacky numbers even though it returns 1 for undefined 👀